articles/200_migration_guide_from_ros1.md

```diff
@@ -102,6 +103,17 @@ Apply the following changes to use `ament_cmake` instead of `catkin`:
 
 * Replace the invocation of `add_message_files`, `add_service_files` and `generate_messages` with [rosidl_generate_interfaces](https://github.com/ros2/rosidl/blob/master/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake).
 
+  * The first argument is the `${PROJECT_NAME}`
```

Comment:
The first argument is the *target name*. Just mentioning `${PROJECT_NAME}` doesn't tell the user anything. The difference is that the user has to choose the target name - were in ROS 1 the function chose the target name for you (without telling you what it is).